test/unit/basic.test.js

```diff
@@ -29,30 +29,34 @@ describe('basic', () => {
       it('should translate simple link', () => {
         assert.equal(i18n.t('message.link'), messages.en.message.hello)
       })
-    })
 
-    describe('linked translation', () => {
       it('should translate link at the end of locale', () => {
         assert.equal(i18n.t('message.linkEnd'), 'This is a linked translation to the world')
       })
-    })
 
-    describe('linked translation', () => {
       it('should translate link within a locale', () => {
         assert.equal(i18n.t('message.linkWithin'), 'Isn\'t the world we live in great?')
       })
-    })
 
-    describe('linked translation', () => {
       it('should translate multiple links within a locale', () => {
         assert.equal(i18n.t('message.linkMultiple'), 'Hello hoge!, isn\'t the world great?')
       })
-    })
 
-    describe('linked translation', () => {
```

Comment:
Note:  Some `describe('linked translation', () => {` lines are consolidated to single one.
I thinkt it is better to describe one concept that has several test cases.

Before 
```
linked translation
 ✔️ should foo
linked translation
 ✔️ should bar
linked translation
 ✔️ should buz
```

After
```
linked translation
 ✔️ should foo
 ✔️ should bar
 ✔️ should buz
```